src/look_transform.rs

```diff
@@ -25,15 +25,26 @@ pub struct LookTransformBundle {
 pub struct LookTransform {
     pub eye: Vec3,
     pub target: Vec3,
+    pub up: Vec3,
```

Comment:
I'm really not sure this is required though. I pushed this then realized your system seems Y-axis-as-up locked. It that's indeed the case, I should remove the possibility to modify the "up vector".
I'm might have been a bit confused, as people often do when playing with cameras... Basically I'm trying to reproduce Unity's flyby camera, which is very similar to Unreal, but allows the developer to rotate _past_ the XY plane, like a "space ship", but without the roll.